components/dynatrace_activegate_private/nonprod.tf

```diff
@@ -0,0 +1,16 @@
+module "nonprod_dynatrace_activegate_private" {
+  source                  = "../../modules/dynatrace-activegate_private"
+  instance_count          = 2
+  dynatrace_instance_name = "yrk32651"
+  env                     = "nonprod"
+  # network_zone            = "azure.cft"
```

Comment:
is there a network zone or not?

Reply:
Want to test it without - will cleanup later, documentation didn't reference it but kept here just to confirm